### PR TITLE
remote tools listed at the end #issue

### PR DESCRIPTION
**Pull request by ashishkamra**
1. collect all remote tools while going through the list of tools at /var/lib/pbench-agent
2. add the list of remote tools to the tools list at the end of the loop
3. still the tools are listed on per group basis. Not sure if that is ok or do we want remote tools from ALL groups listed at the end?

### CODE REVIEW

File: agent/util-scripts/pbench-list-tools [hunk 1]
```diff
@@ -106,6 +112,7 @@ if [ -d "$pbench_run" ]; then
 					tools="$tools,$this_tool_file"
                         	fi
 			done
+			tools="$tools,$all_remote_tools"
```
Comment (ndokos) [inline agent/util-scripts/pbench-list-tools]: If there are no remote tools, we get a comma at the end of the output.

**Comment by ndokos** [comment]
For question 3), we *do* want each group listed separately: groups do not mix.

**Comment by atheurer** [comment]
I agree with Nick, groups are not meant to be mixed.  Output should be for a specific group requested, or the default group

File: agent/util-scripts/pbench-list-tools (hunk 1 above)
Comment (ashishkamra) [inline agent/util-scripts/pbench-list-tools]: will fix that

**Comment by ashishkamra** [comment]
Fixes #633

File: agent/util-scripts/pbench-postprocess-tools [hunk 2]
```diff
@@ -192,6 +192,7 @@ for this_tool_file in `/bin/ls $tool_group_dir`; do
 	fi
 done
 for p in $pids ;do
+        echo "stopping pid: $p"
```
Comment (portante) [inline agent/util-scripts/pbench-postprocess-tools]: Looks like this line is prefixed by spaces and not tabs.

File: agent/util-scripts/unittests [hunk 3]
```diff
@@ -197,6 +197,7 @@ declare -A tools=(
     [test-28]="pbench-collect-sysinfo"
     [test-29]="pbench-collect-sysinfo"
     [test-30]="pbench-collect-sysinfo"
+    [test-31]="pbench-list-tools"
```
Comment (portante) [inline agent/util-scripts/unittests]: Are there other files not provided here that enable this unit test?

**Comment by portante** [comment]
#1248 obviates this work.

**Pull request closed without merging**